tests/fxl-spread_both/OPS/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>When rendition:spread is "both", the content should appear in a synthetic spread regardless of reading system orientation.</dc:description>
+    <dc:identifier id="pub-id">fxl-spread_both</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>rendition:spread both</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#def-spread-both</meta>
+    <meta property="dcterms:modified">2021-11-24T00:00:00Z</meta>
+    <meta property="rendition:layout">pre-paginated</meta>
+    <meta property="rendition:spread">both</meta>
+    <meta property="belongs-to-collection">should</meta>
```

Comment:
Nit: wherever it appears, belongs-to-collection should be alphabetized above the other <meta> tags.